ruby/lib/minitest/queue/failure_formatter.rb

```diff
@@ -29,6 +29,7 @@ def to_h
           test_name: test.name,
           test_suite: test.klass,
           error_class: test.failure.exception.class.name,
+          error_message: test.failure.exception.message,
```

Comment:
I would include the backtrace in the error_message as well, since our users would want to know that information as well. You can reuse the code from line 63 in this file for this, just make sure to remove the terminal colours.